src/serializers/schema.js

```diff
@@ -31,6 +31,7 @@ function SchemaSerializer() {
         'enums',
         'integration',
         'isFilterable',
+        'isPrimaryKey',
```

Comment:
Actually wondering if the `isPrimaryKey` property is serialized to `true` or `false` for any field as it is for other properties (and if not, if it should)

Reply:
@arnaudbesnier I don't understand the question.

Do you want to know if there is always a value set for this property, on every field?

For what I remember, properties like `isRequired` is only set to true when a field is required, and not set at all when a field is not.

I did the same for `isPrimaryKey`, but this behavior is part of both `forest-express-sequelize` and `forest-express-mongoose`, not this package.

Reply:
Ok, I changed the code to always set the property